### PR TITLE
enhance: add db label for deletion and bulkinsert operation

### PR DESCRIPTION
**Pull request by jaime0815**

### CODE REVIEW

File: internal/datacoord/import_scheduler.go [hunk 1]
```diff
@@ -287,7 +287,15 @@ func (s *importScheduler) processInProgressImport(task ImportTask) {
 			log.Warn("update import segment rows failed", WrapTaskLog(task, zap.Error(err))...)
 			return
 		}
+
+		collInfo := s.meta.GetCollection(task.GetCollectionID())
```
Comment (bigsheeper) [inline internal/datacoord/import_scheduler.go:291]: please get the dbName from outside the for loop

**Comment by jaime0815** [comment]
rerun ut

**Review by LoveEachDay — APPROVED** [review]
/lgtm

**Comment by jaime0815** [comment]
rerun ut

**Review by LoveEachDay — APPROVED** [review]
/lgtm

**Pull request merged**